src/content/tutorials/en/intro-to-p5-strands.mdx

```diff
@@ -0,0 +1,744 @@
+---
+title: "p5.strands: Introduction to Shaders"
+description: An introduction to shaders using p5.strands
+category: p5-strands
+categoryIndex: 0
+featuredImage: ../images/featured/intro-to-p5-strands.jpg
+featuredImageAlt: An abstract, cosmic scene with particles
+relatedContent:
+  references:
+    - en/p5/p5shader
+    - en/p5shader/modify
+    - en/p5shader/inspecthooks
+    - en/p5/basecolorshader/
+    - en/p5/basematerialshader/
+    - en/p5/basestrokeshader/
+    - en/p5/basenormalshader/
+    - en/p5/basefiltershader/
+authors:
+  - Luke Plowden
+---
+
+import EditableSketch from "../../../components/EditableSketch/index.astro";
+import Callout from "../../../components/Callout/index.astro";
+
+## Introduction
+
+In this tutorial, we'll explore **p5.strands**, a new way of writing shaders using JavaScript in p5.js. Shaders are programs that run on the GPU to create visual effects that would be difficult to achieve with traditional rendering.

```

Comment:
Minor suggestion:

"Shaders are programs that run on the GPU to create visual effects that would be difficult to achieve with traditional rendering." -> "Shaders are programs that run in parallel on the GPU to create visual effects that would be difficult to achieve with traditional rendering, which run in sequence on the CPU."

Maybe it can be part of the following paragraph? I think it's important to at least partly state in the introduction _how_ they are more efficient. And for similar reason mentioning CPU: "run on GPU" - as opposed to what?